universal-login-example/src/views/DevicesModalView.js

```diff
@@ -1,7 +1,7 @@
 import React, { Component } from 'react';
```

Comment:
I feel we are overusing this modal. This whole code can be perfectly replaced by a "confirm()" javascript call, which is not only less code but more native. The current code doesn't look well on mobile.

<img width="535" alt="screen shot 2018-10-10 at 2 01 52 pm" src="https://user-images.githubusercontent.com/112898/46753013-120f8300-cc95-11e8-9672-6bbe0a85917e.png">

Reply:
Could you expand on this?